### ISSUE TITLE
Server and Database tab of Mirth Connect Server Manager does not display correctly.

### ISSUE THREAD

**Issue by rbeckman-nextgen** [nextgenhealthcare/connect#3071]
When you fill in the jdbc connection string in url field on the database tab 
and the url is rather long then the display will not be shown correctly when you switch between the Server tab and Database tab 

jdbc:sqlserver://ServerName.fullyqualified.DomainName.net;instanceName=instancename;databaseName=mirth_database;integratedSecurity=true;

Imported Issue. Original Details:
Jira Issue Key: MIRTH-3159
Reporter: scholtst
Created: 2014-02-26T00:10:14.000-0800

**Comment by rbeckman-nextgen** [comment]
1 and 2 is when a large url is filled
3 and 4 is when the url is emptied.

Imported Comment. Original Details:
Author: scholtst
Created: 2014-02-26T00:16:04.000-0800

**Comment by rbeckman-nextgen** [comment]
Using an older build, I populated the URL field in the MC Manager with an arbitrarily long path. After switching between tabs, it appears the UI grew outside of the bounds of the frame dimensions. I was unable to replicate this behavior in 3.1 in the same manner. Verified the UI is static regardless of how big or small the path is.

Imported Comment. Original Details:
Author: jaysenp
Created: 2014-08-20T14:48:21.000-0700
